avocado/utils/lv_utils.py

```diff
@@ -0,0 +1,517 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Harish <harisrir@linux.vnet.ibm.com>
+# Copyright: 2016 IBM
+#
+# Based on the code by
+# Author: Plamen Dimitrov
+# Copyright: Intra2net AG 2012
+
+"""
+Utility for taking shapshots from existing Logical volumes
+or creates such.
+
+:param vg_name: Name of the volume group.
+:param lv_name: Name of the logical volume.
+:param lv_size: Size of the logical volume as string in the form "#G"
+        (for example 30G).
+:param lv_snapshot_name: Name of the snapshot with origin the logical
+        volume.
+:param lv_snapshot_size: Size of the snapshot with origin the logical
+        volume also as "#G".
+:param ramdisk_vg_size: Size of the ramdisk virtual group.
+:param ramdisk_basedir: Base directory for the ramdisk sparse file.
+:param ramdisk_sparse_filename: Name of the ramdisk sparse file.
+:param disk: Name of the disk to create volume group. If not provided,
+             creates loop disk and creates volume group on it.
```

Comment:
Would you please move those docstrings to each method docstring instead? That way it gets included in API documentation automatically.